File: benchmarks/comparison/fib.js
function fib(n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

console.log("NodeJS: Starting Fib(25)...");
const start = Date.now();
const res = fib(25);
const end = Date.now();
console.log("NodeJS: Result = " + res);
console.log("NodeJS: Time = " + (end - start) + "ms");

File: benchmarks/comparison/fib.py
import time

def fib(n):
    if n < 2: return n
    return fib(n-1) + fib(n-2)

print("Python: Starting Fib(25)...")
start = time.time() * 1000
res = fib(25)
end = time.time() * 1000
print(f"Python: Result = {res}")
print(f"Python: Time = {end - start:.2f}ms")
